### ISSUE TITLE
3.1.3: There is no simple way of achieving what's described in 3.1.3

### ISSUE THREAD

**Issue by kidayasuo** [w3c/jlreq#164]
To achieve the requirement described in 3.1.3 you need to be able to tell if the IDEOGRAPHIC COMMA / KATAKANA MIDDLE DOT placed among ideographic numbers are intended as thousands separator / decimal point. It can not be achieved by just looking at adjacent characters.

Considering the complexity of achieving the requirement, and its relative importance, we might want to remove this feature from the "requirements". Precise wording is to be discussed.

from 2020-01-30 TF meeting.

**Comment by xfq** [comment]
This feature does not necessarily need to be automatically detected by the browser or e-book reader. Content authors should be able to mark it up using HTML classes and use CSS to control the presentation (i.e., spacing). I think that jlreq should not be limited to include only typographic features that the browser/RS can automatically detect and implement. For example, we can use something like this:

```html
<style>
/* CSS does not have this property value. I just use trim-decimal-point as a possible example. */
.ideographic-numeral {
  text-spacing: trim-decimal-point;
}
</style>
<span class="ideographic-numeral">〇・三五一四</span>
```

**Comment by kidayasuo** [comment]
Right. Probably we can explain with some examples that this feature can't be automated and that global application of the style can have side effects.

**Comment by macnmm** [comment]
What was the use case that broke this? The ambiguity of 〇? Or when • appears at line start?

**Comment by kidayasuo** [comment]
e.g. 「二、三人」

**Comment by kojiishi** [comment]
Opt-in sounds good to me. I think the opt-in should be an OpenType feature so that font designer can specify how much spacing they want for different cases.
